src/main/java/ExecutorServiceSolutionCallableImpl.java

```diff
@@ -0,0 +1,23 @@
+import java.util.List;
+import java.util.concurrent.Callable;
+
+public class ExecutorServiceSolutionCallableImpl implements Callable<Integer> {
```

Comment:
callable is not necessarily connected to ExecutorService
```suggestion
public class SumCalculatorCallable implements Callable<Integer> {
```